nix/overlay.nix

```diff
@@ -1,4 +1,7 @@
-self: super: {
+self:
+let helm-mapkubeapis = self.callPackage ./pkgs/helm-mapkubeapis.nix { };
```

Comment:
I was a bit surprised seeing this expression here.

Have you tried to move it closer to its usage? E.g. to line 16?

Reply:
I did. But then it gave me "variable helm-mapkubeapis unknown" or something similar. When I moved it up here it worked.

Reply:
It's not that bad. So, never mind :shrug: :smile: